presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1861,4 +1862,17 @@ public HiveClientConfig setLegacyTimestampBucketing(boolean legacyTimestampBucke
         this.legacyTimestampBucketing = legacyTimestampBucketing;
         return this;
     }
+
+    @Config("hive.orc-use-vector-filter")
+    @ConfigDescription("Experimental: enable vector path for orc filters")
```

Comment:
note that this requires jdk 19. Also, would probably remove "Experimental".  Notes like that tend to just stick around forever. Would be great to add validation that the jdk meets the minimum version and fail with invalid_configuration_property otherwise.